modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/collection.json

```
@@ -0,0 +1,4 @@
+{
+	"description": "",
+	"name": "EVP"
```

Could you change the name to:
```suggestion
	"name": "EVP Fragments"
```